Target new API endpoint `/extra_files/raw`

mvdbeek opened the pull request:
As the vitessce webclient will append relative paths to the base url

Fixes empty panels that make use of anndata files:

![image](https://github.com/user-attachments/assets/5c191a0e-ac80-4946-a50b-3584d061d719)

This is because vitessce frontend will build urls like https://usegalaxy.org/api/datasets/f9cad7b01a4721358531a44ecfb1aff6/display/obs/.zattrs?filename=A/1/bdbd683b-1870-4c11-ae29-b35ff0be59b2.adata.zarr where we would (previously) need https://usegalaxy.org/api/datasets/f9cad7b01a4721358531a44ecfb1aff6/display?filename=A/1/bdbd683b-1870-4c11-ae29-b35ff0be59b2.adata.zarr/obs/.zattrs

I've added another endpoint at `api/datasets/{dataset_id}/extra_files/raw/{path}`, so that the vitessce frontend logic will work correctly.

### Please replace this header with a short description of your pull request. Please include BOTH what you did and why you made the changes

---

**This PR is related to**
- [ ] Adding a new tool 
- [ ] Updating an existing tool to a newer version
- [ ] Fixing a bug or updating just the Galaxy wrapper of an existing tool
- [ ] Making a change to the `tools-mti` repo, CI, or other misc. change

---

**If updating an existing tool to a newer major version**
- [ ] I have updated the `TOOL_VERSION` token in the tool's macros file 
- [ ] I have reset the `VERSION_SUFFIX` to `0` in the tool's macros file 

---

### Provide details here

- Cite relevant [issues](https://github.com/goeckslab/tools-mti/issues) if applicable
- If fixing a bug, please add any relevant error or traceback
- If adding or updating tools, please describe how tool requirements are satisfied (Examples: "Created new Docker container for scimap", "Added scimap to Bioconda", etc.)

mvdbeek committed:
Target new API endoint /extra_files/raw

As the vitessce webclient will append relative paths to the base url

luke-c-sargent commented:
heya, thanks for the changes!

since this uses [a new API endpoint](https://github.com/galaxyproject/galaxy/pull/20468), does this mean this version of the tool will only work for galaxy `v25.0` and up (e.g. unlikely to be back-ported since its not a bugfix)? should we then bump the `@PROFILE@` token from `22.01` to `25.0`?

luke-c-sargent committed:
set profile version string in main_macros.xml 

as this feature depends on an API endpoint in 25.0, the tool should require that version

luke-c-sargent reviewed (approved):
tested on galaxy.ohsu.edu, works a treat
